src/main/resources/application.yml

```diff
@@ -25,6 +25,7 @@ client:
   fhir-terminology-service:
     base-url: ${FHIR_TERMINOLOGY_BASE_URL:https://uat-cts.nlm.nih.gov/fhir}
     manifests-urn: /Library
+    code-system-urn: /res/CodeSystem
```

Comment:
I think  the `/res/` might be non-standard. It does work, but it also works with out. I'll need to investigate a bit to see which way better matches the spec.

Reply:
I will resolve for now

Reply:
`/res/` is non-standard and VSAC supports requests both with and without it. SME's are thinking it could represent the URL is part of a Response. Though, neither of us are sure what VSAC uses it for.